src/types.rs

```diff
@@ -1,6 +1,10 @@
 use crate::errors;
 use std::result;
 
+// A time tag in OSC message consists of a two 32-bit integers where the first one specifies the number of seconds since 1900-01-01 and the second fractions of a second.
+// For details on its semantics see http://opensoundcontrol.org/node/3/#timetags
+pub type OscTime = (u32, u32);
+
```

Comment:
it seems like you should use the triple slash, `///` so you get documentation of this type?

Reply:
Ah, right, I wrote so much Go in the mean time that I forgot about the triple slash.